### ISSUE TITLE
Tooltip displaying seconds instead of ns/day

### ISSUE THREAD

**Issue by cxhernandez** [rmcgibbo/openmm-bench#4]
This is super minor, but I think if we hack https://github.com/rmcgibbo/asv/blob/master/asv/www/asv.js#L72, we could get the benchmark results to display correctly.

**Comment by rmcgibbo** [comment]
Yeah, I set the attribute on the tests described at the very bottom of the page here, http://asv.readthedocs.org/en/latest/writing_benchmarks.html, but i don't see it show up on the ylabel of the graphs (or in the tooltip). It looks like they're developing pretty actively, so we can fix it and PR.
